packages/desktop-client/src/components/budget/BalanceWithCarryover.tsx

```diff
@@ -135,31 +137,31 @@ export function BalanceWithCarryover({
                   <span style={{ color: theme.noticeText }}>Fully funded</span>
                 ) : differenceToGoal > 0 ? (
                   <span style={{ color: theme.noticeText }}>
-                    Overfunded ({format(differenceToGoal, 'financial')})
+                    {t('Overfunded')} ({format(differenceToGoal, 'financial')})
```

Comment:
Here, I think the amount needs to be part of the translation string.  So something like this:

```suggestion
                    {t('Overfunded ({{amount}})', { amount: format(differenceToGoal, 'financial') })}
```

Reply:
Yes, or maybe in that language parens don't make sense.